project/profiles/capacity4more/modules/c4m/ds/c4m_ds/includes/c4m_ds.theme.user.inc

```diff
@@ -106,3 +108,21 @@ function theme_c4m_ds_user_footer_stats(array $field) {
 
   return implode($items);
 }
+
+/**
+ * DS function field.
+ *
+ * Gets the my group block.
+ *
+ * @param array $field
+ *   The field info.
+ *
+ * @return string
+ *   The rendered field.
+ */
+function theme_c4m_ds_user_my_groups(array $field) {
+  $block = block_load('c4m_user_profile', 'user_my_profile_groups');
+  $renderable_blocks = _block_render_blocks(array($block));
+  $renderable_array = _block_get_renderable_array($renderable_blocks);
+  return render($renderable_array);
```

Comment:
@DavidHernandez 
add a line spacing above this one.
(you can add `[ci skip]` to your commit message in order to skip travis)